visualization/create_charts.py

```
@@ -0,0 +1,366 @@
+            # 베이스라인보다 높으면 녹색, 낮으면 빨간색
+            if baseline_acc is not None and acc >= baseline_acc:
+                colors.append('green')
+            else:
+                colors.append('red')
+
+        # 플롯 생성
+        plt.figure(figsize=(10, 6))
+        bars = plt.barh(methods, accuracies, color=colors, alpha=0.7)
+
+        # 수치 표시
+        for bar, acc in zip(bars, accuracies):
```

Ruff (0.14.13)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)